retrieval/company_info.py

```diff
@@ -124,6 +124,10 @@ def company_number(self) -> str:
     def company_url(self) -> str:
         return self._company_url
     
+    @property
+    def links(self) -> dict:
+        return self._links
```

Comment:
You've put self._links here (which is good) so you need it to be the same when you declare it above

Reply:
cool. fixed. self.links -> self._links in lines.